packages/material-ui-unstyled/src/BadgeUnstyled/BadgeUnstyled.test.js

```diff
@@ -0,0 +1,199 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createMount, createClientRender, describeConformance } from 'test/utils';
+import BadgeUnstyled from './BadgeUnstyled';
+import classes from './badgeClasses';
```

Comment:
Prefer the public API

```suggestion
import BadgeUnstyled, {badgeClasses } from '@material-ui/unstyled/BadgeUnstyled';
```